activerecord/test/cases/fixtures_test.rb

```diff
@@ -1589,3 +1591,19 @@ def readonly_config
       end
   end
 end
+
+class StrictLoadingFixturesTest < ActiveRecord::TestCase
+  fixtures :strict_zines
+
+  test "strict loading violations are ignored on fixtures" do
+    ActiveRecord::FixtureSet.reset_cache
+    create_fixtures("strict_zines")
+
+    strict_zines(:going_out).interests.to_a
```

Comment:
Can you assert the contents of this array or remove it? Then you don't need the comment below.